### ISSUE TITLE
2.1.17 Cannot finde module './dist/sassdoc'

### ISSUE THREAD

**Issue by gisu** [SassDoc/sassdoc#440]
Sassdoc 2.1.17 drops Error. 2.1.15 works fine.

```bash
module.js:339
    throw err;
    ^

Error: Cannot find module './dist/sassdoc'
    at Function.Module._resolveFilename (module.js:337:15)
    at Function.Module._load (module.js:287:25)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/Users/gisu/Development/private/webdevs-craft/node_modules/sassdoc/index.js:3:15)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Module.require (module.js:366:17)
```

**Comment by JonasFromell** [comment]
Came here for this. The dist folder is missing alltogether, which is why this error is showing up

**Comment by MediaAlex** [comment]
Have the same issue...

**Comment by valeriangalliat** [comment]
Should be fixed with 2.1.18
